docs/conf/portland/2021/news/announcing-speakers.rst

```diff
@@ -0,0 +1,49 @@
+:template: {{year}}/generic.html
+
+.. post:: February 14, 2021
+   :tags: portland-2021, speakers, tickets, visiting, sponsors
+
+Announcing speakers and sessions
+================================
+
+We're writing today with an update on the fast-approaching Portland conference!
+
+First off, we're delighted to announce that we've finalized our speakers for the event!
+Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
+We're really excited about the presentations we've got this year – we hope you are too!
+
+.. datatemplate::
+   :source: /_data/{{shortcode}}-{{year}}-sessions.yaml
+   :template: {{year}}/speakers-simple-list.rst
+
+Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts.
+We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
+
+Tickets are going fast!
+-----------------------
+
+If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have sold out for the last several years, and they go faster along with the speaker announcement, so if you're planning to come, don't leave it to the last minute.
+
+`Buy your tickets here! <https://www.writethedocs.org/conf/portland/{{year}}/tickets/>`_
+
```

Comment:
I'd add the copy about t-shirt sales here (we should have the text in last year's post), and also turn on the t-shirt flag in the yaml file.

Link to this year's shirts (notice the URL format is different in case it matters):
https://teespring.com/wtd-pdx-2021